config/routes.rb

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-
   get '/auth/:provider/callback' => 'sessions#create'
   delete '/signout' => 'sessions#destroy', :as => :signout
   get '/auth/failure' => 'sessions#failure'
+  resource :session, only: [:create, :destroy]
```

Comment:
? why is this needed? we're not using devise anymore

Reply:
I know, but when I was trying to Fix the SessionController spec `post` request was broken with a UrlGenerationError like this:

```
 ActionController::UrlGenerationError:
       No route matches {:action=>"create", :controller=>"sessions"}
```

that's why I added on `routes` not sure if there is another approach to this